hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -141,4 +147,18 @@ public void frameworkMessage(ExecutorDriver driver, byte[] msg) {
         .build());
     }
   }
+
```

Comment:
it is the zkfc that is very different.   when the launch is called for NN, the ZK node is also launched.  This is not the case for any other node.   perhaps we need to discuss it.